server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -125,6 +129,10 @@ public Invocations getInvocations() {
             .invocation().methods(GET).path("/v2/container/config").withAction("listen")
                .permission(AuthorizationPermission.ADMIN).auditContext(AuditContext.SERVER).handleWith(this::listenConfig)
 
+            // Container lifecycle listener
+            .invocation().method(GET).path("/v2/container").withAction("listen")
+               .permission(AuthorizationPermission.ADMIN).auditContext(AuditContext.SERVER).handleWith(this::listenLifecycle)
```

Comment:
Are the config events supposed to be sent/received in this endpoint?

Reply:
so why create a new endpoint? why not just create a new parameter in `config` stream to include the lifecycle events?

Reply:
The reason for a new endpoint is to reflect that this isn't specific to config, but container events in general. It's not intuitive for a stream of config+lifecycle events to be consumed from a `config` sub-resource i.e .`/v2/container/config`

Reply:
well, it is confusing to me. If I want only lifecycle events, I have to use `/v2/container?action=listen` and filter out the events I don't want...  
If we add view change (and xsite view change) events, would they go into this new endpoint? or create a new one?

Reply:
I think all events should be exposed via `/v2/container`. There's no need to create a new endpoint for the events you describe. I agree that some kind of server-side filtering would be nice to allow only requested events to be sent over the wire. We could add support for filtering via parameters, with only requested event types being returned if one or more filters are applied, the default being to send everything if no filters are applied.

Reply:
+100 to filtering

Reply:
WDYT @jabolina?

This PR has been open for a while now, so I think it's acceptable to merge in it's current state (once the SecurityAction changes have been applied) and create a Jira to add event filtering in a subsequent PR.